### PR TITLE
Bsweger/mkerr research page updates

### PR DESCRIPTION
**Pull request by bsweger**
@mmkerr @nickreich This is an test to fix diagnose the PR that failed the Netlify build: https://github.com/reichlab/reichlab.github.io/pull/190

No need to review yet--just trying to see what the Netlify bot does with this PR!

### CODE REVIEW

**Comment by bsweger** [comment]
FWIW, this isn't a Netlify issue--I'm getting the same Ruby error when building the site locally (it happens when generating content via the research plugin).

Am taking a closer look--not sure why this would have stopped working suddenly!

File: _plugins/research.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def generate(site)
             end
           end
         }
-        if fm.key? 'publications'
+        if fm.key? 'publications' && !fm['publications'].nil? && !fm['publications'].empty?
```
Comment (bsweger) [inline _plugins/research.rb:32]: This line fixes the broken build by checking for an empty/null value in the "publications" part of the front-matter before attempting to parse

Comment (nickreich) [reply _plugins/research.rb:32]: I'm confused why this was throwing an error. where was there no publications list?

Comment (nickreich) [reply _plugins/research.rb:32]: That said, it does seem to work now.

File: _research/02-hubs.md
Comment (nickreich) [inline _research/02-hubs.md:1]: @mmkerr In the preview build, the hubverse logo image is too large vertically, so there is wasted space. Can you trim the logo so it is fit to the size that it is?

File: _research/02-hubs.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Infrastructure for Modeling Hubs
+image: /images/research/modeling-hubs.png
+projects:
+  -  hubverse-org/hubDocs
```
Comment (nickreich) [inline _research/02-hubs.md:5]: @mmkerr can you add some of the other hubverse-org repos here? We could add all of them, but maybe simpler to just add a few key ones, how about:

- hubverse-org/hubverse
- hubverse-org/hubData
- hubverse-org/hubEnsembles

**Review by nickreich — CHANGES_REQUESTED** [review]
A few small cosmetic changes requested.

**Comment by mmkerr** [comment]
See PR #206 in response to Nick's comments (sorry, I made a new branch since I didn't know if I should make changes to Becky's branch)

**Pull request merged**